django_mongodb/expressions.py

```diff
@@ -13,7 +15,22 @@ def value_agg(self, compiler, connection):  # noqa: ARG001
     return {"$literal": self.value}
 
 
+def expression(self, compiler, connection):
+    return self.expression.as_mql(compiler, connection)
+
+
+def expression_agg(self, compiler, connection):
+    return self.expression.as_mql_agg(compiler, connection)
+
+
+def cast(self, compiler, connection):  # noqa: ARG001
+    raise NotSupportedError("Cast is not supported on this database backend")
+
+
 def register_expressions():
     Col.as_mql = col
     Value.as_mql = value
     Value.as_mql_agg = value_agg
+    ExpressionWrapper.as_mql = expression
+    ExpressionWrapper.as_mql_agg = expression_agg
+    Cast.as_mql = cast
```

Comment:
Cast is implemented in https://github.com/mongodb-labs/django-mongodb/pull/46. I'd appreciate your review.